pkgs/development/python-modules/scienceplots/default.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  matplotlib,
+}:
+
+buildPythonPackage rec {
+  pname = "SciencePlots";
+  version = "2.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-2NGX40EPh+va0LnCZeqrWWCU+wgtlxI+g19rwygAq1Q=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ matplotlib ];
+
```

```suggestion

  pythonImportsCheck = [ "scienceplots" ];

```
This will ensure the package can be imported.